src/main/java/com/atlassian/bitbucket/jenkins/internal/link/BitbucketExternalLinkUtils.java

```diff
@@ -0,0 +1,74 @@
+package com.atlassian.bitbucket.jenkins.internal.link;
+
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketServerConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMRepository;
+import hudson.util.FormValidation;
+import okhttp3.HttpUrl;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Objects;
+import java.util.Optional;
+
+@Singleton
+public class BitbucketExternalLinkUtils {
+
+    @Inject
+    private BitbucketPluginConfiguration bitbucketPluginConfiguration;
+
+    @Inject
```

Comment:
do we need `@Inject` twice for effectively the same field? It is legal but it seems undesirable?